container/container.go

```
@@ -490,9 +495,9 @@ func copyEscapable(dst io.Writer, src io.ReadCloser, keys []byte) (written int64
 				}
 				if i == len(keys)-1 {
 					if err := src.Close(); err != nil {
-						return 0, err
+						return 0, true, err
```

we don't really need to report this error